blemmy/feed/models.py

```diff
@@ -0,0 +1,43 @@
+from django.db import models
+from wagtail.wagtailcore.models import Page
+from wagtail.wagtailcore.fields import RichTextField
+from wagtail.wagtailadmin.edit_handlers import FieldPanel, StreamFieldPanel, InlinePanel, MultiFieldPanel, FieldRowPanel
+from wagtail.wagtailimages.edit_handlers import ImageChooserPanel
+from wagtail.wagtailsearch import index
+# API model
+from wagtail.api import APIField
+
+# Create your models here.
+
+class FeedPage(Page):
+    intro = RichTextField(blank=True)
+
+    main_image = models.ForeignKey(
+        'wagtailimages.Image',
+        null=True, blank=True,
+        on_delete=models.SET_NULL,
+        related_name='+'
+    )
+    postdate = models.DateField(default='YYYY-MM-DD', blank=True)
+    body = RichTextField(blank=True)
+
+    api_fields = [
```

Comment:
Why did you comment out the other API fields? If you don't need these fields, don't include them in your model at all. In a headless CMS *all* fields should be exposed.